src/middlewares/authenticator.js

```diff
@@ -19,33 +19,35 @@ class Authenticator {
   }
 
   /**
-   * @method verifyToken
+   * @method authenticateUser
    * @description Verifies that user is authenticated
    * @param {object} req - The Request Object
    * @param {object} res - The Response Object
    * @param {object} next - The next function
    * @returns {object} - JSON response object
    */
-  static async verifyToken(req, res, next) {
+  static async authenticateUser(req, res, next) {
     let token = '';
     /* eslint-disable no-unused-vars */
```

Comment:
I will not support this.

Reply:
@Temmyogunbo Could you please suggest a type of naming for this function because I named according to the convention 'Authenticating Users'